example/android/app/build.gradle

```diff
@@ -22,7 +22,7 @@ if (flutterVersionName == null) {
 }
 
 apply plugin: 'com.android.application'
-apply plugin: 'com.google.gms.google-services'
+//apply plugin: 'com.google.gms.google-services'
```

Comment:
Is it worth removing this, or should it stay commented out?

Reply:
These are all push notification dependencies. The example app wound't work with those items. However, I have to check what is the issue as I was able to integrate push to FoxFacts.